### DISCUSSION TITLE
Felles sporingsid på tvers av aktørene

### DISCUSSION THREAD

**Discussion by micer2004** [NorskHelsenett/Tillitsrammeverk#57] [Ideas]
Refererer til diskusjon i tirsdagsmøte 14. mars 2023 hvor tematikken om "gjennomgående korrelasjons" ble tatt opp.

En kort oppsummering av møte er følgende:
- det finnes mange use case/behov som kan understøtte behovet for bruk av felles sporings ID (både funksjonell og teknisk)
- det bør ses i sammenheng med andre behov på tvers av nasjonale tjenester slik at man unngår duplisering av funksjonalitet
- inkludering av sporings ID i datamodellen for tillitsrammeverk er ikke ønskelig men bruk av token/headers foretrekkes. I dokumentdeling foregår utveksling fra JWT til SAML, noe som må ivaretas.
- dette bør ses i sammenheng med innbyggerens innsynsrett og modellering av funksjonalitet som skal tilrettelegges for behovet der

**Comment by richardhus** [comment]
Det bør også sees i sammenheng med loggoppfølging som følger av tilganger mellom virksomheter generelt. Og jeg tenker det er litt for tidlig å konkludere på at dette ikke bør inn i informasjonsmodellen for tillitsrammeverket siden det fort er veldig aktuelt å ha en ID som gjelder all aktivitet i kontekst av en dokumentert behandlerrelasjon mellom en pasient og helsepersonellet, da mener jeg dette er en veldig naturlig og enkel utvidelse å legge her. Men for tekniske sporingsformål og feilsøking blir det helt feil, selv om en slik identifikator også sikkert kan bidra noe til det formålet, særlig om man har kobling mellom dem (i et en-til-mange forhold nødvendigvis) logget.

**Reply by troelde** [reply]
@richardhus : Hvis dette er det samme som `hso:subject:application-session:id` , så er denne analog til en vanlig sesjonsid som er utstedt ved brukerpålogging til konsumentens EPJ, og endres ikke fra pasient til pasient som velges aktivert/aktualisert i EPJ.
